.circleci/config.yml

```diff
@@ -264,7 +264,7 @@ jobs:
       _JAVA_OPTIONS: "-Xmx3g"
       GRADLE_OPTS: "-Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2"
     docker:
-      - image: cimg/openjdk:11.0.16-node
+      - image: cimg/openjdk:19.0.2-node
```

Comment:
shouldn't we use 17 here as well though?

Reply:
Makes sense!